wealth-management-client/src/app/reports/userviews/userview-grid.component.html

```diff
@@ -0,0 +1,10 @@
+<p-dataTable [immutable]="false" [value]="userInvestmentList" [sortMode]="multiple" [responsive]="true">
```

Comment:
portfolio.component.html

Reply:
achivedPercentage will be called Profit/Loss and shall be calculated with investment price.
targetPercentage will be calculated with investment price.

Reply:
achivedPercentage based on the recommended view.
Looks like the understanding of the US is different between us.
Lets have a call and sync-up.

Reply:
Profit/Loss is specific to user view.

targetPercentage/achivedPercentage is based on the recommendation.

Reply:
Changes based on our sync-up